setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="turn-python",
-    version="0.2.0",
+    version="0.2.1",
```

Comment:
[Semantic versioning](https://semver.org/) follows this structure:
<major>.<minor>.<patch>. See the link for what each of these means. Also see my comment about remoding the `send_audio` method